toolchain/ltrace.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
```

Comment:
Btw @jayarama-ibm I'm searching the test in `autotest-client-tests` and I can't find it there. So I guess it was internal one, right?